src/components/CoinCard/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useRef, useState } from 'react';
+import cancelClose from '../../assets/svg/cross_cancel_icon.svg';
+import styles from './CoinCard.module.scss';
+import { useNavigate, useParams } from 'react-router-dom';
+import { getCoin } from '../../api/api';
+import { CoinDetailedInfo } from '../../types/types';
+import Loader from '../Loader';
+
+const {
+  card,
+  card__name,
+  card__rank,
+  card__price,
+  card__symbol,
+  card__icon,
+  card__description,
+  card__change,
+  card__close,
+} = styles;
```

Comment:
That's a lot of imports, I'd say it would be simpler to just use the classes as e.g. `styles.card`, `styles.card__name`, etc.